layouts/search-index/single.html

```diff
@@ -0,0 +1,5 @@
+{{- $.Scratch.Add "index" slice -}}
```

Comment:
@LisaFC: I can't make it work. What other changes did you apply to the files?

Reply:
So my version that works has
(in the Docsy theme)
layouts/search-index/single.html
then in the userguide I have
userguide/content/en/search-index.md with the following contents:

---
type: "search-index"
url: "index.json"
---

plus I've enabled your param in the userguide's config.toml

offlineSearch = true

Try it and let me know if it works!

Reply:
any luck?

Reply:
Works like a charm. Uploading the changes in a minute.

Reply:
It works. :) We'll have to add it to the documentation I guess? Or make it a default file of the repo.

Reply:
Yep, let's add it to the docs (I'm happy to do that once everything is checked in) and then add a search-index.md to both the userguide in this repo and the docsy-example site.